docs/getting-started/my-account/README.md

```diff
@@ -91,36 +91,25 @@ You can select the order of the comments (for example of the comments for a work
 
 If you choose newest first the latest comment will appear on top in the Activity list.
 
-![order-to-display-comments](order-to-display-comments.png)
+![my-account-settings-comments](order-to-display-comments.png)
 
 ### Backlogs settings
 
 There are two personal settings available for the [Backlogs module](../../user-guide/backlogs-scrum):
-**Task color**: Here you can enter the color in which your tasks on the task board (on the Backlogs page) are displayed. Tasks on the task board are color-coded based on who they are assigned to. Every user has a default color assigned but if you prefer a different color, you can change it here.
+**Task color**: Here you can enter the color in which your tasks on the task board (on the Backlogs page) are displayed. Tasks on the task board are color-coded based on who they are assigned to. Every user has a default color assigned but if you prefer a different color, you can change it.
 **Show versions folded**: When you navigate to the "Backlogs" module in a project, the available versions (e.g. Product backlog, Sprints) are displayed. By default they are expanded, i.e. all work packages included in those versions are shown right away. If you have a large number of versions that are shown on the "Backlogs" page, you may opt to collapse them by default. This way you see all the versions on a page and can then choose to expand those versions for which you want to see details.
 
 ## Two-factor authentication (Premium feature)
 
 In order to activate the two-factor authentication for your OpenProject installation, navigate to your **My account** and choose the **Two-factor authentication** in the menu.
 
-![Two-factor authentication](1571992032476.png)
+![two-factor-authentication](Bildschirmfoto 2022-03-02 um 14.50.58.png)
 
 In order to register a new device for two-factor authentication, lick the green button to add a **new 2FA device**.
 
-You can either use your mobile phone number to receive the second factor via SMS. Or you can register to use an app-based authenticator, such as Google Authenticator or Authy.
+To receive the second factor, you can use an authentication app on your mobile phone, such as Google Authenticator or Authy. You have to enter the code that is displaed in the authentication app to your login.
 
-![add new 2fa device](1572876883531.png)
-
-### Register your mobile phone number for delivery of OpenProject one-time passwords.
-
-To register your phone number to receive a one time password via SMS, click on the **Register device** button under the **Use your mobile phone** section.
-
-- Choose an identifier how you want your phone to be named.
-- Enter your phone number in the following format.
-
-![two-factor authentication](1572878991958.png)
-
-Press the blue **Continue** button to confirm your device.
+![app-based-autentication](Bildschirmfoto 2022-03-02 um 15.43.40.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.